sqlmesh/migrations/v0019_add_schema_to_data_hash.py

```diff
@@ -0,0 +1,5 @@
+"""Add schema to data_hash (https://github.com/TobikoData/sqlmesh/pull/1212)"""
+
+
+def migrate(state_sync):  # type: ignore
```

Comment:
I think the next release will be breaking anyway (0.23.0). Do we still need this script then?

Reply:
Ah, I guess we do unless we also update sqlglot version